__tests__/port.test.js

```diff
@@ -1,13 +1,35 @@
-const Port = require('../src/port.js')
+const Port = require('../src/port')
+describe ('Port', () => {
+    let manchester;
+    let ship;
+    let david;
+    let alpha;
 
+beforeEach(() => {
+    manchester = new Port ('Manchester');
+    ship = jest.fn();
+    david = jest.fn();
+    alpha = jest.fn();
+})
+describe('constructor', () => {
+    it('instantiates an object', () => {
+        expect(new Port('Manchester')).toBeInstanceOf(Object);
+    })
+    
+    it('sets a name property', () => {
+        expect(manchester.name).toEqual('Manchester');
+    })
 
-describe('Port', () => {
-    it('instantiates a port object', () => {
-        expect(new Port()).toBeInstanceOf(Object);
+    it('adds ships to a port', () => {
+        manchester.addShip(ship);
+        expect(manchester.ships).toContain(ship);
     })
 
-    it('sets the name property', () => {
-        const port = new Port('Korea');
-        expect(port.name).toEqual('Korea');
+    it('removes ships from a port', () => {  
+        manchester.addShip(david);
+        manchester.addShip(alpha);
+        manchester.removeShip(alpha);
+        expect(manchester.ships).toEqual([david]);
     })
-});
\ No newline at end of file
+})
```

Comment:
as in `ship.test.js`, the `beforeEach...` and `descibe...` block should be indented by one